tests/wpt-harness/tests.json

```
@@ -163,6 +163,18 @@
   "fetch/api/response/response-stream-with-broken-then.any.js",
   "fetch/data-urls/base64.any.js",
   "fetch/data-urls/processing.any.js",
+  "FileApi/blob/Blob-array-buffer.any.js",
+  "FileApi/blob/Blob-bytes.any.js",
+  "FileApi/blob/Blob-constructor.any.js",
+  "FileApi/blob/Blob-constructor-dom.window.js",
+  "FileApi/blob/Blob-constructor-endings.html",
+  "FileApi/blob/Blob-in-worker.worker.js",
+  "FileApi/blob/Blob-slice.any.js",
+  "FileApi/blob/Blob-slice-overflow.any.js",
+  "FileApi/blob/Blob-stream.any.js",
+  "FileApi/blob/Blob-stream-byob-crash.html",
+  "FileApi/blob/Blob-stream-sync-xhr-crash.html",
```

We don't support running HTML tests:

```suggestion
  "FileApi/blob/Blob-in-worker.worker.js",
  "FileApi/blob/Blob-slice.any.js",
  "FileApi/blob/Blob-slice-overflow.any.js",
  "FileApi/blob/Blob-stream.any.js",
```

---

Thanks. It turned out the file path is `FileAPI` not `FileApi` 🤦 I fixed it with this:

https://github.com/bytecodealliance/StarlingMonkey/pull/169/commits/701606692384fd60f4f4863e6b751bd604eca94e